Jenkinsfile_CNP

```diff
@@ -8,7 +8,7 @@ import uk.gov.hmcts.contino.GithubAPI
 def type = "java"
 def product = "civil"
 def component = "service"
-def ccdBranch = "master"
+def ccdBranch = "CIV-8281_hearing_times_to_be_auto_populated_with_default_dates"
```

Comment:
Before you merge this PR, please revert back this change.

Reply:
sure